sdk/attestation/attestation/sample.env

```diff
@@ -1,5 +1,6 @@
 AAD_ATTESTATION_URL="<AAD attestation URL>"
 ISOLATED_ATTESTATION_URL="<Isolated attestaion URL>"
+ATTESTATION_LOCATION_SHORT_NAME="<Short name of the location of the test instance>"
 policySigningCertificate2=""
```

Comment:
I see those two in the karma config:
```
"ATTESTATION_ISOLATED_SIGNING_CERTIFICATE",
"ATTESTATION_ISOLATED_SIGNING_KEY",
```
should they be added here or removed from there?

Reply:
As far as I can tell, they need to be in both places (actually, I don't think anything strictly needs to be in sample.env, except as documentation).

Reply:
Yup, let's add them here then as well with a comment on how to set them so future contributors to this package can test it.